src/js/media-time-display.js

```diff
@@ -106,6 +108,9 @@ class MediaTimeDisplay extends MediaTextDisplay {
       'background',
       'var(--media-control-hover-background, rgba(50 50 70 / .7))'
     );
+    const { style: noToggle } = getOrInsertCSSRule(this.shadowRoot, ':host([notoggle])');
+    noToggle.setProperty('cursor', 'default');
+    noToggle.setProperty('background', 'var(--media-text-background, var(--media-control-background, var(--media-secondary-color, rgb(20 20 30 / .7))))');
```

Comment:
could this line be removed if hover selector was changed to `:host(:hover:not([notoggle]))` or equivalent?
save some bytes

Reply:
ah that's a good idea. I think both styles can probably be applied using the same logic, actually.